### PR TITLE
Fix starting osbuild-composer

### PR DESCRIPTION
**Pull request by bcl**
This is based on PR #1007 from @atodorov. and changes should be made to that PR. Do not merge this one.

### CODE REVIEW

**Comment by bcl** [comment]
For some reason the TestImages.test_compose_sanity is failing with 'tar' missing from the list it checks. This was removed way back in 2019. No idea why it wasn't noticed before...

File: test/composertest.py [hunk 1]
```diff
@@ -135,6 +136,19 @@ def setUp(self):
...
+        curl_command = ["curl", "--max-time", "360",
+                                "--silent",
+                                "--unix-socket", "/run/weldr/api.socket",
+                                "http://localhost/api/status"]
+        r = subprocess.run(self.ssh_command + curl_command, stdout=subprocess.PIPE)
+        self.assertEqual(r.returncode, 0)
+        status = json.loads(r.stdout)
+        print(status)
+        self.assertTrue("backend" in status)
+        self.assertEqual(os.getenv("BACKEND"), status["backend"])
```
Comment (atodorov) [inline test/composertest.py:150]: @bcl I'm not sure I understand this block. It duplicates the curl_command and almost duplicates subprocess.run. Did you mean to execute this twice or simply forgot to delete the old block ?

**Comment by atodorov** [comment]
@bcl for the fedora32/osbuild-composer test job I still see lorax-composer as the backend in the logs.

File: test/composertest.py (hunk 1 above)
Comment (bcl) [reply test/composertest.py:150]: Yeah, didn't clean up things after I got it working. Except that in #1007 it looks like the return code is 20 for some reason, and curl doesn't document what 20 means...

**Pull request closed without merging**